### PR TITLE
[Merged by Bors] - Validator import password flag

### PR DESCRIPTION
**Pull request by realbigsean**
## Issue Addressed

#2224

## Proposed Changes

Add a `--password-file` option to the `lighthouse account validator import` command. The flag requires `--reuse-password` and will copy the password over to the `validator_definitions.yml` file. I used #2070 as a guide for validating the password as UTF-8 and stripping newlines.

### CODE REVIEW

File: account_manager/src/validator/import.rs [hunk 1]
```diff
@@ -65,13 +65,28 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
...
                 .help("If present, the same password will be used for all imported keystores."),
         )
+        .arg(
+            Arg::with_name(PASSWORD_FLAG)
+                .long(PASSWORD_FLAG)
+                .value_name("KEYSTORE_PASSWORD_PATH")
+                .requires(REUSE_PASSWORD_FLAG)
+                .help(
+                    "The path to the file containing the password which will unlock all \
+                    keystores being imported. This flag must be used with the `--reuse-password`. \
```
Comment (paulhauner) [inline account_manager/src/validator/import.rs]:
```suggestion
                    keystores being imported. This flag must be used with `--reuse-password`. \
```

**Review by paulhauner — CHANGES_REQUESTED** [review]
Nice one! Just one little typo.


Also, it might be pretty easy and valuable to duplicate this test and get it to use a password file:

https://github.com/sigp/lighthouse/blob/46920a84e857d6c086adf4b42810ac03de94ed3e/lighthouse/tests/account_manager.rs#L407-L497

**Comment by realbigsean** [comment]
Comments addressed!

**Review by paulhauner — APPROVED** [review]
Nice one!

bors r+

**Comment by paulhauner** [comment]
@realbigsean seems bors doesn't like this, potentially for valid reasons. It might be worth merging `unstable` and then seeing if it passes CI.

**Comment by realbigsean** [comment]
got a green check!

**Review by paulhauner — APPROVED** [review]
Awesome!

bors r+

**Comment by paulhauner** [comment]
I'll batch this

bors r-

**Pull request closed without merging**